src/metrics/definitions.md

```diff
@@ -0,0 +1,9 @@
+# Metric Definitions
+
+It is important to understand that metrics are often defined in terms of a usage criterion.
+
+* A `usage criterion` defines what a metric is measuring.  It generally specifies a product and what specific use of that product qualifies as active.  For example, “Any Fenix Activity” measures activity for all Fenix profiles that have sent a `baseline` ping.  Similarly, “Opened DevTools” means that we measure activity for all desktop Firefox profiles that have sent a telemetry ping indicating that DevTools was opened.
+
+* A `dimension` allows slicing to a subset of profiles according to characteristics of those profiles.  Available dimensions include: country, channel, OS, etc.  A slice is a set of particular values within dimensions, for example, “country is US” is one slice and “country is either US or DE and channel is release” is another.
+
+* A `metric` summarizes activity for a particular day.  It is defined in terms of a `usage criterion` and `slice`.  For example, the `DAU` metric will tell you how many profiles met the `usage criterion` within the `slice` or, more concretely, might tell you how many profiles had any Firefox Desktop activity within Canada.
```

Comment:
> A `metric` summarizes activity for a particular day.

The first few times I read this, I interpreted it as summarising activity _on_ a particular day, which is not the case. A metric can summarise activity over some time period (e.g. 28 days for MAU), and we define that time period with respect to a particular day. Is there some way to make this clearer?

Reply:
As part of the above definition, I've completely changed to metric definition.  It probably needs further work though, so I'm open to suggestions.